shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/compiler/BaseOliveBuilder.java

```diff
@@ -105,6 +105,37 @@ public BaseOliveBuilder(RootBuilder owner, int oliveId, Type initialType) {
 
 	}
 
+	/**
+	 * Create a “Matches” clause in an olive
+	 *
+	 * @param matcher
+	 *            the matcher to run
+	 * @param arguments
+	 *            the arguments to pass as parameters to the matcher
+	 */
+	public final void call(OliveDefineBuilder matcher, Stream<Consumer<Renderer>> arguments) {
```

Comment:
Since the `Matches` keyword got taken out, maybe update the `matcher` terminology to something related to `Define`?

Reply:
Fixed.